tools/network-scene-vis.md

```diff
@@ -0,0 +1,50 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
```

Comment:
Small nit, I think the tool is just called "Network Scene Visualization" rather than "the Network Scene Visualization". At least that's how I've always heard it referred to internally for the past 9 months 🙂